### PR TITLE
Add LangtonAnt task Behabior to merge dev/langtonant

### PR DESCRIPTION
**Pull request by clopezbaya**

### CODE REVIEW

File: katas/langtonant/christian/movements.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import { make2DArray } from './matrix.js';
+
+
+export var grid;
+export var x;
+export var y;
+export var dir;
+
+//estados de la hormiga
```
Comment (J0c4) [inline katas/langtonant/christian/movements.js:9]: lets use english

File: katas/langtonant/christian/movements.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+//Function move Ant
+export function move(size, steps) {
+    for (let n = 0; n < steps; n++) {
+        let state = grid[x][y];
+        if (isWhite(state)) {
+            turnRight();
+            grid[x][y] = '1';
+        } else if (isBlack(state)) {
+            turnLeft();
+            grid[x][y] = '';
```
Comment (J0c4) [inline katas/langtonant/christian/movements.js:74]: does `''` represent a white block? if so, what would 0 represent?

**Review by J0c4 — CHANGES_REQUESTED** [review]
This looks good, there are a couple of lint issues that need to be fixed and also, the empty file for tests is causing tests to file, please add tests for movements.test.js

File: katas/langtonant/christian/movements.js (hunk 1 above)
Comment (clopezbaya) [reply katas/langtonant/christian/movements.js:9]: ok

**Review by J0c4 — APPROVED** [review]

**Pull request merged**